packages/protocol/tipmanager/tipmanager.go

```diff
@@ -141,7 +147,7 @@ func (t *TipManager) RemoveStrongParents(block *models.Block) {
 // RemoveStrongParents removes all tips that are parents of the given block.
 func (t *TipManager) removeStrongParents(block *models.Block) {
 	block.ForEachParent(func(parent models.Parent) {
-		// TODO: reintroduce TipsConflictTracker
+		// TODO: we remove the tip, but we reintroduce the pending liked conflict through the earliest attachment.
```

Comment:
This bit is actually obsolete: the new conflicttracker rescues conflicts, not specific tips.